src/components/ResourceItem/ResourceItem.tsx

```diff
@@ -342,6 +343,12 @@ class BaseResourceItem extends React.Component<CombinedProps, State> {
     this.setState({focused: false, focusedInner: false});
   };
 
+  private handleMouseOut = () => {
+    if (this.state.focused) {
+      this.setState({focused: false, focusedInner: false});
+    }
```

Comment:
`this.state.focused && this.setState({focused: false, focusedInner: false});` should fix the code coverage. We're not executing the else branch so this bypasses it. It doesn't really matter since we don't have any code after this.